targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/device/TOOLCHAIN_ARM_STD/MK64FN1M0xxx12.sct

```diff
@@ -1,4 +1,4 @@
-#! armcc -E
+#! armclang -E --target=arm-arm-none-eabi -x c -mcpu=cortex-m4
```

Comment:
I had an error earlier because of the old armcc there but cant reproduce o nthe recent branch. I'll keep this in and once we have armclang6 linking, cna revert this one if that is the case.